package.json

```diff
@@ -25,9 +25,15 @@
     "@actions/exec": "^1.0.1",
     "@actions/io": "^1.0.1",
     "@actions/tool-cache": "^1.1.2",
-    "valid-url": "^1.0.9"
+    "eslint-config-airbnb-base": "^14.2.1",
```

Comment:
this should not have to be here

Reply:
I think I did this by mistake, trying something else